addons/payment/security/ir.model.access.csv

```diff
@@ -4,7 +4,8 @@ payment_provider_onboarding_wizard,payment.provider.onboarding.wizard,model_paym
 payment_provider_system,payment.provider.system,model_payment_provider,base.group_system,1,1,1,1
 payment_icon_all,payment.icon.all,model_payment_icon,,1,0,0,0
 payment_icon_system,payment.icon.system,model_payment_icon,base.group_system,1,1,1,1
-payment_token_all,payment.token.all,model_payment_token,,1,0,0,0
```

Comment:
You don't need to change that. Please see [our documentation](https://www.odoo.com/documentation/master/developer/reference/backend/security.html#access-rights).

Reply:
after discussion because the public user still need access, this is indeed unnecessary, thanks :)